src/components/pages/pitch-arena/PitchNavigation.tsx

```
@@ -0,0 +1,236 @@
+          </div>
+
+          {/* CENTER: Logo */}
+          
+          <div className="absolute left-[48%] top-1/2 -translate-y-1/2 pointer-events-none">
+
+
+            <button
+              onClick={onHome}
+              aria-label="Go to home"
+              className="pointer-events-auto"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)